### PR TITLE
Created French Translation

### PR DESCRIPTION
**Pull request by rene-coty**
I created fr.po files + updated the LINGUAS files

### CODE REVIEW

File: po/extra/fr.po [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+msgid "Individually change each audio source's values"
+msgstr "Changez individuellement les valeurs de chaque source audio"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:19
+msgid "Allie Law"
+msgstr "Allie Law"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:32
+msgid "Nothing new!"
+msgstr "Rien de nouveau !"
```
Comment (ChildishGiant) [inline po/extra/fr.po:65]: Is the space at the end needed?

File: po/fr.po [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: src/MainWindow.vala:26 src/MainWindow.vala:39
+msgid "Mixer"
+msgstr "Mixeur"
+
+#: src/MainWindow.vala:210
+msgid "Volume:"
+msgstr "Volume :"
```
Comment (ChildishGiant) [inline po/fr.po:27]: More spaces after words

File: po/extra/fr.po [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+"alphabétiques"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:58
+msgid "Updated AppCenter metadata"
+msgstr "Métadonnées de l’AppCenter mises à jour"
+
+#: data/com.github.childishgiant.mixer.appdata.xml.in:63
+#: data/com.github.childishgiant.mixer.appdata.xml.in:72
+msgid "Added features:"
+msgstr "Fonctionnalités ajoutées :"
```
Comment (ChildishGiant) [inline po/extra/fr.po:130]: 👾 There's an invading space!

File: po/extra/fr.po [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-12-24 21:21+0000\n"
+"PO-Revision-Date: 2022-06-20 17:15+0200\n"
+"Last-Translator: \n"
```
Comment (ChildishGiant) [inline po/extra/fr.po:12]: Feel free to add your name here

**Review by ChildishGiant — COMMENTED** [review]
A few little tweaks then I'll happily merge

File: po/extra/fr.po (hunk 1 above)
Comment (rene-coty) [reply po/extra/fr.po:65]: Yes, in French there is always a space between words and some punctuation signs like «, », !, ?, ; ...

File: po/fr.po (hunk 2 above)
Comment (rene-coty) [reply po/fr.po:27]: It's normal ;)

**Review by ChildishGiant — APPROVED** [review]

**Comment by ChildishGiant** [comment]
@all-contributors please add @rene-coty for translation

**Pull request merged**